pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-facebook/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, pidgin, glib, json_glib } :
+
+stdenv.mkDerivation rec {
+  name = "purple-facebook-${version}";
+  version = "66ee77378d82";
+
+  src = fetchurl {
+    url = "https://github.com/dequis/purple-facebook/releases/download/${version}/purple-facebook-${version}.tar.gz";
+    sha256 = "0m34blh99fmsqanhi65zp3mn5aqhlbfmmqy28aph0mr98sy20i5w";
+  };
+
+  makeFlags = [
+    "PLUGIN_DIR_PURPLE=/lib/pidgin/"
+    "DATA_ROOT_DIR_PURPLE=/share"
+    "DESTDIR=$(out)"
+  ];
+
+  postInstall =  ''
+    mkdir -p $out/lib/purple-2
+    cp pidgin/libpurple/protocols/facebook/.libs/*.so $out/lib/purple-2/
+  '';
+
+  buildInputs = [ pidgin glib json_glib ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/dequis/purple-facebook;
```

Comment:
If you use `fetchFromGitHub` you could use `inherit (src.meta) homepage;` here :wink: